dc/s2n-quic-dc/src/stream/server/application.rs

```diff
@@ -0,0 +1,251 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    event,
+    stream::{
+        application::Builder as StreamBuilder,
+        environment::{
+            tokio::{self as env, Environment},
+            Environment as _,
+        },
+        runtime::tokio as runtime,
+        server::{
+            accept, stats,
+            tokio::{common_builder_methods, uds::Worker},
+        },
+        socket,
+    },
+    sync::mpmc,
+};
+use core::num::{NonZeroU16, NonZeroUsize};
+use s2n_quic_core::ensure;
+use std::{
+    io,
+    net::SocketAddr,
+    path::{Path, PathBuf},
+};
+use tracing::Instrument as _;
+
+#[derive(Clone)]
+pub struct Server<S: event::Subscriber + Clone> {
+    streams: accept::Receiver<S>,
+    local_addr: SocketAddr,
+    stats: stats::Sender,
+    /// This field retains a reference to the runtime being used
+    #[allow(dead_code)]
+    env: Environment<S>,
+    #[allow(dead_code)]
+    acceptor_rt: runtime::Shared<S>,
```

Comment:
Why do we have the allow(dead_code) here? Are these only kept for a Drop impl or something?

Reply:
Removed those fields here, but I'm not sure why we have it in the [original server](https://github.com/aws/s2n-quic/blob/main/dc/s2n-quic-dc/src/stream/server/tokio.rs#L64) in the first place. fmt/clippy/build do not seem to complain without it.